homeassistant/components/insteon.py

```diff
@@ -35,10 +34,23 @@ def setup(hass, config):
         return False
 
     import insteon
+    if config[DOMAIN].get(CONF_USERNAME) is None:
```

Comment:
Please use [validate_config](https://github.com/balloob/home-assistant/blob/dev/homeassistant/helpers/__init__.py#L9) to ensure that the required configuration entries are available.